FrejaEidClient/src/main/java/com/verisec/frejaeid/client/service/SignService.java

```diff
@@ -34,40 +37,57 @@ public SignService(String serverAddress, HttpServiceApi httpService) throws Frej
+            throws FrejaEidClientInternalException, FrejaEidException {
+        MethodUrl methodUrl = transactionContext == TransactionContext.ORGANISATIONAL ?
+                MethodUrl.ORGANISATION_SIGN_INIT : MethodUrl.SIGN_INIT;
+        return httpService.send(getUrl(serverAddress, methodUrl), RequestTemplate.INIT_SIGN_TEMPLATE, signRequest,
+                                InitiateSignResponse.class, signRequest.getRelyingPartyId());
     }
 
-    public SignResult getResult(SignResultRequest signResultRequest) throws FrejaEidClientInternalException, FrejaEidException {
-        MethodUrl methodUrl = transactionContext == TransactionContext.ORGANISATIONAL ? MethodUrl.ORGANISATION_SIGN_GET_ONE_RESULT : MethodUrl.SIGN_GET_RESULT;
-        return httpService.send(getUrl(serverAddress, methodUrl), RequestTemplate.SIGN_RESULT_TEMPLATE, signResultRequest, SignResult.class, signResultRequest.getRelyingPartyId());
+    public SignResult getResult(SignResultRequest signResultRequest)
+            throws FrejaEidClientInternalException, FrejaEidException {
+        MethodUrl methodUrl = transactionContext == TransactionContext.ORGANISATIONAL ?
+                MethodUrl.ORGANISATION_SIGN_GET_ONE_RESULT : MethodUrl.SIGN_GET_RESULT;
+        return httpService.send(getUrl(serverAddress, methodUrl), RequestTemplate.SIGN_RESULT_TEMPLATE,
+                                signResultRequest, SignResult.class, signResultRequest.getRelyingPartyId());
     }
 
-    public SignResult pollForResult(SignResultRequest signResultRequest, int maxWaitingTimeInSec) throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
+    public SignResult pollForResult(SignResultRequest signResultRequest, int maxWaitingTimeInSec)
+            throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
         long pollingEndTime = System.currentTimeMillis() + TimeUnit.SECONDS.toMillis(maxWaitingTimeInSec);
-        while (maxWaitingTimeInSec == 0 || ((System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime)) {
+        while (maxWaitingTimeInSec == 0
+                || ((System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime)) {
             SignResult getSignResult = getResult(signResultRequest);
             if (maxWaitingTimeInSec == 0 || isFinalStatus(getSignResult.getStatus())) {
                 return getSignResult;
             }
             try {
                 Thread.sleep(pollingTimeoutInMilliseconds);
             } catch (InterruptedException ex) {
-                throw new FrejaEidClientInternalException(String.format("An error occured while waiting to make another request with %ss polling timeout.", maxWaitingTimeInSec), ex);
+                throw new FrejaEidClientInternalException(
+                        String.format("An error occured while waiting to make another request with %ss polling " +
```

Comment:
Could we resolve this one word problem somehow? :)

Reply:
Only by rephrasing the error message. Is that okay?

Reply:
Correction, I managed it without changing the text.